extensions/azurePublish/yarn.lock

```diff
@@ -1308,6 +1308,7 @@
     "@emotion/core" "^10.0.27"
     "@emotion/styled" "^10.0.27"
     office-ui-fabric-react "7.71.0"
+    react-markdown "^5.0.3"
```

Comment:
I don't think we need react-markdown.  I didn't end up using it in the new copy on the UI as the designs wanted some precise positioning.

Reply:
I think this is appearing because react-markdown is part of the handoff instructions widget.